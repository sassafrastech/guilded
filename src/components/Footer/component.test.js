import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

jest.mock('./Partners/component', () => 'Partners');

it('renders', () => {
  const tree = renderer.create(<Component />);
  expect(tree.toJSON()).toMatchSnapshot();
});
