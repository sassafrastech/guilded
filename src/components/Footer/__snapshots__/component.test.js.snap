// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="makeStyles-container-1"
>
  <Partners />
  <p
    className="MuiTypography-root makeStyles-legalText-2 MuiTypography-body2"
  >
    Copyright Â© 2021
  </p>
</div>
`;
